.github/workflows/integration.yml

```
@@ -10,12 +10,7 @@ jobs:
     strategy:
       matrix:
         os: [Ubuntu, MacOS, Windows]
-        python-version: [3.7, 3.8, 3.9]
-        experimental: [false]
-        include:
-          - os: Ubuntu
-            python-version: "3.10.0-alpha - 3.10.0"
-            experimental: true
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

just needs to be quoted (else it's interpreted as the float `3.1`):
```suggestion
        python-version: [3.7, 3.8, 3.9, "3.10"]
```

---

Good catch!